<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CENTAUR_NAMESPACE::LogDialog</class>
 <widget class="QDialog" name="CENTAUR_NAMESPACE::LogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CENTAUR_NAMESPACE::ResizableFrame" name="logMainFrame">
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: qlineargradient(x1:0.5, y1: 0, x2:0.5, y2:1, 
							stop: .1 rgb(23,31,38), stop: 0 rgb(36,44,50) stop: .2  rgb(3,12,19));

border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-left: 1px solid rgb(92,92,92);
border-top: 1px solid rgb(104,104,104);
border-right: 1px solid rgb(104,104,104);
border-bottom: 1px solid rgb(92,92,92);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CENTAUR_NAMESPACE::MovableFrame" name="titleFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color: qlineargradient(x1:0.5, y1: 0, x2:0.5, y2:1, 
							stop: 1  rgb(23,31,38), stop: 0 rgb(36,44,50));
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
border: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="titleFrameLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="titleBarFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
background-color: rgba(0,0,0,0);
border: 0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="titleBarFrameLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="titleBarFrame_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
background-color: rgba(0,0,0,0);
border: 0px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="titleBarFrameLayout2_2">
               <property name="spacing">
                <number>-1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="systemButtonsFrame">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{
background-color: rgba(0,0,0,0);
border: 0px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="systemButtonsFrameLayout_2">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>1</number>
                  </property>
                  <property name="rightMargin">
                   <number>1</number>
                  </property>
                  <property name="bottomMargin">
                   <number>1</number>
                  </property>
                  <item>
                   <widget class="CENTAUR_NAMESPACE::SystemPushButton" name="closeButton">
                    <property name="minimumSize">
                     <size>
                      <width>13</width>
                      <height>13</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>13</width>
                      <height>13</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
border: 0px;
border-radius: 6px;
background-color: rgb(230,80,2);
}

QPushButton:hover{
border: 0px;
border-radius: 6px;
background-color:  rgb(232,94,22);
}

QPushButton:pressed{
border: 0px;
border-radius: 6px;
background-color: rgb(238,136,83);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="titleLabel_2">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Logs</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CENTAUR_NAMESPACE::MovableFrame" name="mainTableFrame">
        <property name="styleSheet">
         <string notr="true">QFrame{
border: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="logsTable">
           <property name="styleSheet">
            <string notr="true">QTableView {
    color: white;
    background-color: rgb(46,53,59);
    alternate-background-color: rgb(61,68,73);
    border: 0px;
}
QTableView QTableCornerButton::section {

   background-color: qlineargradient(x1:0.5, y1: 0, x2:0.5, y2:1, 
							stop: 0 rgb(82,153,212) stop: 1  rgb(91,158,214));
    border: 0px;
}
QTableView::item {
    border: 0px;
    height: 25px;
}
QTableView::item:hover{
    background-color: rgb(79,85,90);
}
QTableView::item:hover:selected{
    background-color: rgb(104,109,113);
	font-weight: 800;
}
QTableView::item:selected{
    color: white;
	font-weight: 600;
    background-color: rgba(91,97,102,125);
}
QHeaderView::section:checked
{
    background-color:  #3387CD;
    color: white;
}
/***************************************************************************/
QHeaderView::section {
    font: normal 800 13px Roboto;
    padding-left: 4px;
    border: 0px;
    height: 26px;
	background-color: qlineargradient(x1:0.5, y1: 0, x2:0.5, y2:1, 
							stop: 0 rgb(13,113,195) stop: 1  rgb(0,105,192));
    color: white;
    padding-left: 4px;
    border: 0px;
}


QHeaderView::section:hover
{
    background-color: qlineargradient(x1:0.5, y1: 0, x2:0.5, y2:1, 
							stop: 0 rgb(82,153,212) stop: 1  rgb(91,158,214));
    color: white;
}


QLineEdit{
font-size: 14px;
border: 0px;
color: rgb(225,225,255);
background-color: #dfdfdf;
border-radius:  0px;
border-bottom: 2px solid  rgb(63,70,76);
min-height: 25px;
padding: 0 0 0 5;
}


QLineEdit:hover{
background-color:  rgb(78,85,90);
border-bottom: 2px solid #6ec6ff;
}

QLineEdit:focus{
color: black;
background-color:  #dfdfdf;
border-bottom: 2px solid rgb(33,150,243);
}
</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>6</number>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>21</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>21</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>21</number>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CENTAUR_NAMESPACE::MovableFrame</class>
   <extends>QFrame</extends>
   <header>../../Library/cui/include/MovableFrame.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CENTAUR_NAMESPACE::ResizableFrame</class>
   <extends>QFrame</extends>
   <header>../../Library/cui/include/ResizableFrame.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CENTAUR_NAMESPACE::SystemPushButton</class>
   <extends>QPushButton</extends>
   <header>../../Library/cui/include/SystemPushButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
